### ISSUE TITLE
[BUG] No space present between the projects section and resume section

### ISSUE THREAD

**Issue by forntoh** [BoineeloMoyo/Portfolio#1]
No space present between the projects section and resume section.

<img width="1361" alt="image" src="https://github.com/user-attachments/assets/bcc8c1fa-0eff-4af6-a7fb-0f7cd199f9ba">

**Comment by BoineeloMoyo** [comment]
Yes i noticed that, but this only occures on a large screen, it responds well to smaller devices. Will fix it.

**Comment by forntoh** [comment]
Still open?

**Comment by BoineeloMoyo** [comment]
Thanks for reminding me, i had forgotten about it
